index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Debas Portfolio | Dbus</title>
+</head>
+<body>
+    <main class="container">
+      <header>
+        <nav class="navbar" id="nav">
+            <a href="#" class="logo">Dbus-G'</a>
+            <div class="menu">
+                <a href="#" class="humberger"><i class="fas fa-bars"></i></a>
+            </div>
+        </nav>
+      </header>
+      <section class="headline-section">
+          <div class="headline-content">
+              <h1>Debas G'ngus</h1>
+              <h2>Software Development</h2>
+              <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,  don’t hestiate to contact me. </p>
+          </div>
+          <img src="./image/horizontal-line.png" alt="horizontal line image">
+          <div class="skill">
+              <ul class="lists"><li>Language</li> 
+                  <li>Javascript</li>
+                  <li>Ruby</li>
+                  <li>Html</li>
+                  <li>Css</li>
+              </ul>
+              <ul class="lists">
+                  <li>Frameworks</li>
+                <li>Bootsrap</li>
+                <li>Ruby on Rails</li>
+                <li>Rspec</li>
+                <li>CapyBara</li>
+                <li>Selenium</li>
+            </ul>
+            <ul class="lists">
+                <li>Skills</li>
+                <li>Codekit</li>
+                <li>Github</li>
+                <li>Codepen</li>
+                <li>Gitlav</li>
+                <li>Terminal</li>
+            </ul>
+          </div>
+          <img src="./image/horizontal-line.png" alt="horizontal line image">
+      </section>
```

Comment:
- [ ] There seems to be an issue with the background image of your headline. According to the template, the circles should stretch across the screen but in your design that isn't the case. Please fix this. Look at the image below:

![debas-headline](https://user-images.githubusercontent.com/36542986/138159155-4bdc0666-a5bb-40d1-804e-84af689b19df.jpg)

- [ ] Going by the template, you are missing the headline icons. Please add them. **Note:** they should be wrapped in a `<ul>` tag with each of them being a link wrapped in a `<li>`

![debas-icons](https://user-images.githubusercontent.com/36542986/138159664-37bb975f-8ca5-4463-9da6-462b9335ddff.jpg)